src/core_plugins/elasticsearch/lib/create_admin_cluster.js

```diff
@@ -0,0 +1,19 @@
+export default function (server) {
+  const config = server.config();
+  const Logger = server.plugins.elasticsearch.ElasticsearchClientLogging;
+
+  class AdminClientLogging extends Logger {
```

Comment:
@epixa, I agree we can export this separately.

@jbudz, since we use `ElasticsearchClientLogging` in other plugins I preferred to expose it on the plugin rather than importing. The usage here was to keep it consistent but I agree we can switch to an import.